examples/rnn_baselines/train.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
```

Comment:
we need to start making a general train program *now*. lets do it after you get this PR in